base/sparse/sparsevector.jl

```diff
@@ -817,7 +817,7 @@ vec(x::AbstractSparseVector) = x
 copy(x::AbstractSparseVector) =
     SparseVector(length(x), copy(nonzeroinds(x)), copy(nonzeros(x)))
 
-function reinterpret{T,Tv}(::Type{T}, x::AbstractSparseVector{Tv})
+function reinterpret(::Type{T}, x::AbstractSparseVector{Tv}) where {T,Tv}
```

Comment:
Along the lines of my general comment above, elsewhere in this commit long-form method definitions with two type parameters consistently use the `where T where Tv` form. Perhaps either this signature should change to `where T where Tv`, or the other long-form method definitions with two type parameters should switch to the compact form?